crates/sui-graphql-e2e-tests/tests/call/owned_objects.exp

```diff
@@ -73,33 +73,22 @@ Response: {
   }
 }
 
-task 8 'run-graphql'. lines 105-123:
+task 8 'run-graphql'. lines 112-130:
 Response: {
   "data": {
     "address": {
-      "objectConnection": {
-        "edges": [
-          {
-            "node": {
-              "owner": {
-                "__typename": "AddressOwner",
-                "owner": {
-                  "address": "0x0000000000000000000000000000000000000000000000000000000000000042"
-                }
-              }
-            }
-          }
-        ]
+      "objects": {
```

Comment:
This is object filter intersection in action for a query for objects owned by an address, where the `owner` field has been further restricted in the `filter`.  Before we ignored the further restriction, now we compute the intersection (which is empty).